apps/admin-ui/src/spa/recurring-reservations/application-rounds/[id]/review/RejectedOccurrencesTable.tsx

```diff
@@ -0,0 +1,189 @@
+    key: "application_id,application_event_id",
+    transform: ({ pk, applicationPk }: RejectedOccurrencesView) =>
+      `${applicationPk}-${pk}`,
+  },
+  {
+    headerTKey: "ApplicationEvent.headings.customer",
+    isSortable: true,
+    key: "applicant",
+    transform: ({
+      applicantName,
+      applicationPk,
+      pk,
+    }: RejectedOccurrencesView) => (
+      <ExternalTableLink
+        href={getApplicationUrl(applicationPk, pk)}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        {truncate(applicantName ?? "-", applicantTruncateLen)}
+        <IconLinkExternal size="xs" aria-hidden />
+      </ExternalTableLink>
+    ),
+  },
+  {
+    headerTKey: "ApplicationEventSchedules.headings.eventName",
+    isSortable: true,
+    key: "rejected_event_name_fi",
+    transform: ({ name }: RejectedOccurrencesView) => {
+      return <span>{truncate(name ?? "-", unitsTruncateLen)}</span>;
+    },
+  },
+  {
+    headerTKey: "ApplicationEvent.headings.unit",
+    isSortable: true,
+    key: "rejected_unit_name_fi",
+    transform: ({ unitName }: RejectedOccurrencesView) => {
+      return <span>{truncate(unitName ?? "-", unitsTruncateLen)}</span>;
+    },
+  },
+  {
+    headerTKey: "ApplicationEventSchedules.headings.reservationUnit",
+    isSortable: true,
+    key: "rejected_reservation_unit_name_fi",
+    transform: ({ allocatedReservationUnitName }: RejectedOccurrencesView) => {
+      return (
+        <span>
+          {truncate(allocatedReservationUnitName ?? "-", unitsTruncateLen)}
+        </span>
+      );
+    },
+  },
+  {
+    headerTKey: "ApplicationEventSchedules.headings.occurrenceTime",
+    isSortable: true,
+    key: "time_of_occurrence",
+    transform: ({ time, link }: RejectedOccurrencesView) => {
+      if (link !== "") {
+        return <ExternalTableLink href={link}>{time}</ExternalTableLink>;
+      }
+      return <span>{time}</span>;
+    },
+  },
+  {
+    headerTKey: "ApplicationEventSchedules.headings.reason",
+    isSortable: true,
+    key: "rejection_reason",
+    transform: ({ reason }: RejectedOccurrencesView) => <span>{reason}</span>,
+  },
+];
+
+const getColConfig = (t: TFunction) =>
+  COLS.map(({ headerTKey, ...col }) => ({
+    ...col,
+    headerName: t(headerTKey),
+  }));
+export const SORT_KEYS = COLS.filter((c) => c.isSortable).map((c) => c.key);
+
+function RejectedOccurrencesTable({
+  rejectedOccurrences,
+  isLoading,
+  sort,
+  sortChanged: onSortChanged,
+}: Props) {
+  const { t } = useTranslation();
+
+  const rows = rejectedOccurrences.map((ro) => timeSlotMapper(t, ro));
+  const cols = memoize(() => getColConfig(t))();
+
+  if (rows.length === 0) {
+    const name = t("ApplicationEvent.emptyFilterPageName");
+    return <div>{t("common.noFilteredResults", { name })}</div>;
+  }
+
+  const sortField = sort?.replaceAll("-", "") ?? "";
+  const sortDirection = sort?.startsWith("-") ? "desc" : "asc";
+  return (
+    <CustomTable
+      setSort={onSortChanged}
```

Comment:
If this is merged: https://github.com/City-of-Helsinki/tilavarauspalvelu-ui/pull/1256 need to add the `disableKey` here also

Reply:
Ok - but It causes a ilnt error since #1256 isn't merged... but `disableKey` added. Now it needs to be merged before merging this branch.